frameworks/keyed/ember/src/components/jumbotron.gjs

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { getOwner } from '@ember/owner';
+
+export const PaddedButton = <template>
+  <div class="col-sm-6 smallpad">
+    <button
+      type="button"
+      class="btn btn-primary btn-block"
+      ...attributes
+    >{{yield}}</button>
+  </div>
+</template>;
+
+export class Jumbotron extends Component {
+  state = getOwner(this).lookup('service:state');
+
+  <template>
+    <div class="jumbotron">
+      <div class="row">
+        <div class="col-md-6">
+          <h1>Ember (keyed)</h1>
+        </div>
+        <div class="col-md-6">
+          <div class="row">
+            <PaddedButton id="run" onclick={{this.state.create}}>
+              Create 1,000 rows
+            </PaddedButton>
+            <PaddedButton id="runlots" onclick={{this.state.runLots}}>
```

Comment:
getting `NaN` for runLots case 
<img width="1205" alt="image" src="https://github.com/user-attachments/assets/0c56bdf2-7053-467c-8389-e8fed5f26829" />

Reply:
fixed